git.d.ts

```diff
@@ -0,0 +1,390 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
@henoktsegayetoptal I think you also worked on these git definitions?

Reply:
yep I removed the git error codes but yes